test/fixtures/poro.rb

```diff
@@ -222,6 +222,18 @@ def maker
   end
 end
 
+TypedRoleSerializer = Class.new(ActiveModel::Serializer) do
+  type 'my-roles'
+  cache only: [:name], skip_digest: true
+  attributes :id, :name, :description, :slug
+
+  def slug
+    "#{object.name}-#{object.id}"
+  end
+
+  belongs_to :author
+end
```

Comment:
would you mind putting this in the test file?  We're try to no longer add files to poro unless they are for general usage